test/unit/service/jobRoleServiceTest.js

```diff
@@ -0,0 +1,38 @@
+var axios = require('axios');
+var MockAdapter = require('axios-mock-adapter');
+var chai = require('chai');  
+const expect = chai.expect;
+const JobRoleService = require('../../../dist/src/services/jobRoleService');
+const job = {
```

Comment:
What you think about having a line break here?

```suggestion
const JobRoleService = require('../../../dist/src/services/jobRoleService');

const job = {
```